android/src/main/java/com/mparticle/react/MParticlePackage.java

```diff
@@ -21,7 +21,6 @@
         return modules;
     }
 
-    @Override
     public List<Class<? extends JavaScriptModule>> createJSModules() {
```

Comment:
Can you add a comment noting why this method is still here at all? My understanding is that we need to keep it to maintain backwards compatibility with pre-`0.47`, but that might not be obvious to future readers of the code.